pkg/blockstorage/awsefs/awsefs.go

```diff
@@ -430,6 +430,9 @@ func (e *efs) SnapshotDelete(ctx context.Context, snapshot *blockstorage.Snapsho
 	req.SetRecoveryPointArn(snapshot.ID)
 
 	_, err := e.DeleteRecoveryPointWithContext(ctx, req)
+	if isResourceNotFoundException(err) {
+		return nil
+	}
```

Comment:
Clarified with Sirish on a call - This was added to make SnapshotDelete idempotent.